SETUP.md

```diff
@@ -543,16 +499,6 @@ If you run into an error like "Don't know how to set rpath on your system, if My
 
 - `gem install mysql2 -v '0.5.2' -- --with-opt-dir=$(brew --prefix openssl) --with-ldflags=-L/opt/homebrew/Cellar/zstd/1.5.0/lib`
 
-#### therubyracer
-
-If you run into an issue about therubyracer while running `bundle install` try :
```

Comment:
Note that although this PR is not removing `therubyracer`, I'm unable to find any other references to it in the codebase, and based on the linked SO question it seems like this is actually an issue not with `therubyracer` itself but with `libv8`, which this PR _is_ removing.